NMS-13446: provisiond docs updates (#3577)
OpenNMS/opennms: docs/modules/operation/pages/provisioning/adapters/adapters.adoc

[[adapters]]
= Adapters

The {page-component-title} Provisiond API also supports provisioning adapters (plugins) for integration with external systems during the provisioning import phase.

> These adapters allow the exchange of data between OpenNMS and another system, allowing the enrichment of node data.

Author: These adapters let OpenNMS exchange data with another system, allowing the enrichment of node data.
```suggestion
These adapters let OpenNMS exchange data with other systems, allowing the enrichment of node data.
```
